### PR TITLE
add .jscsrc

### PR DESCRIPTION
**Pull request by CCharlieLi**

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "preset": "airbnb",
+    "requirePaddingNewLinesAfterBlocks": false,
+    "requirePaddingNewLinesBeforeLineComments": false,
+    "safeContextKeyword": ["self"],
+    "requireTrailingComma": false,
+    "disallowEmptyBlocks": false
+}
```
Comment (makara) [inline .jscsrc]: - Indent by 2 spaces.
- Sort keys by letter.
- A newline at the file end.

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 !.gitignore
 !.npmignore
 !.jshintrc
```
Comment (makara) [inline .gitignore]: Then we don't need `.jshintrc`. Please remove it.

File: .jscsrc (hunk 1 above)
Comment (CCharlieLi) [inline .jscsrc]: `Indent by 2 spaces.` and `A newline at the file end.` are already preset in `airbnb`.

Comment (makara) [inline .jscsrc]: No I mean for the `.jscsrc` file itself.

**Comment by makara** [comment]
Thanks.

**Pull request merged**